composites/AlgoliaSearch/tests/__snapshots__/SearchResultDetailTest.js.snap

```diff
@@ -8,6 +8,7 @@ exports[`the SearchResultDetail component matches the snapshot 1`] = `
     className="sc-gzVnrw kJqUwD"
   >
     <button
+      aria-expanded={undefined}
```

Comment:
undefined

Reply:
I think this is normal in the snapshots, and just means the prop hasn't been passed. Better ask @Xyfi  though. (there are other similar occurrences in the snapshots)

![screen shot 2017-09-29 at 12 53 22](https://user-images.githubusercontent.com/1682452/31013203-59097228-a515-11e7-8a16-aeb697754094.png)

Reply:
Let me be more specific 😄 : Why isn't the prop passed in the test?
To me it seems to be good practice to not have `undefined` values in the tests, because you wouldn't want `undefined`s in the eventual implementation.

Reply:
First of all, this test is about the Algolia search "back button" which is unrelated to this PR and I haven't touched the test. This button is not supposed to have an aria-expanded attribute (while the Help Center one does!). It inherits from the YoastButtonBase which now has an aria-expanded attribute based on a `isExpanded` prop. In React, the proper way to _not_ render an attribute is to not pass the related prop, i.e. let it be undefined, so React doesn't render the attribute.
What the snapshots do is not equivalent to the actual rendering.
In a few words, the Back Button must not test the aria-expanded attribute because it doesn't have this attribute set. Instead, the Help Center button does need to have the attribute, if you check its snapshot it has `aria-expanded={false}`.

Reply:
Ah, thanks for the explanation :)